coderedcms/templates/wagtailcore/password_required.html

```diff
@@ -0,0 +1,20 @@
+{% extends "coderedcms/pages/web_page_notitle.html" %}
+{% load i18n bootstrap4 %}
+
```

Comment:
Question - does this page have a title that gets rendered in the browser tab? I'm assuming we probably should override the title block here.